managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/MetaStoreImplZookeeper.java

```diff
@@ -233,84 +224,76 @@ public void asyncUpdateCursorInfo(final String ledgerName, final String cursorNa
                 info.getCursorsLedgerId(), info.getMarkDeleteLedgerId(), info.getMarkDeleteEntryId());
 
         String path = prefix + ledgerName + "/" + cursorName;
-        byte[] content = info.toString().getBytes(Encoding);
+        byte[] content = info.toByteArray();
```

Comment:
@merlimat what do we suggest we do then?

I realized broker also saves in ZK the loadbalance info of all bunldes it owns, this can potentially grow very large too, right?